### PR TITLE
Updates to Self-Reflected XSS

### PR DESCRIPTION
**Pull request by RRudder**
* Removal of placeholder text
* revision of overview, business impact and steps to repro

### CODE REVIEW

File: submissions/description/cross_site_scripting_xss/reflected/self/guidance.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<!--
```
Comment (vortexau) [inline submissions/description/cross_site_scripting_xss/reflected/self/guidance.md:1]: If we don't have guidance, can/should this template be removed?

**Review by vortexau — CHANGES_REQUESTED** [review]
Empty guidance template, if this isn't to be included it should probably be removed but if this is intended, lmk and I'll approve.

File: submissions/description/cross_site_scripting_xss/reflected/self/guidance.md (hunk 1 above)
Comment (RRudder) [reply submissions/description/cross_site_scripting_xss/reflected/self/guidance.md]: Thanks for the pickup, this has been removed and the recommendation tweaked with the correct `HTTP` code block added

Comment (RRudder) [reply submissions/description/cross_site_scripting_xss/reflected/self/guidance.md]: Thanks for the pickup, this has been removed and the recommendation tweaked with the correct `HTTP` code block added

**Review by vortexau — APPROVED** [review]

**Pull request merged**